openmdao/core/total_jac.py

```diff
@@ -1207,6 +1212,32 @@ def _do_scaling(self, J):
             raise RuntimeError("Derivative scaling by the driver only supports the 'dict' and "
                                "'array' formats at present.")
 
+    def _print_debug_derivatives(self):
+        """
+        Print out the derivatives when debug_print is True.
+        """
```

Comment:
this is really just `_print_derivatives`... the "when debug_print is True" part is external to this function...

Reply:
renamed